Assets/Scripts/Controllers/SpawnInventoryController.cs

```diff
@@ -86,60 +89,90 @@ private void CreateSpawnUI()
         csf.verticalFit = ContentSizeFitter.FitMode.MinSize;
     }
 
-    private void CreateInventoryButtons()
+    private void CreateInventoryEntries()
     {
         foreach (string invName in PrototypeManager.Inventory.Keys)
         {
-            GameObject inventoryButton_go = new GameObject();
-            inventoryButton_go.name = "Button - " + invName;
-            inventoryButton_go.layer = LayerMask.NameToLayer("UI");
+            GameObject inventorySlot_go = new GameObject();
+            inventorySlot_go.name = "Slot - " + invName;
+            inventorySlot_go.layer = LayerMask.NameToLayer("UI");
+
+            inventorySlot_go.transform.SetParent(spawnUI.transform);
 
-            inventoryButton_go.transform.SetParent(spawnUI.transform);
+            HorizontalLayoutGroup hlg = inventorySlot_go.AddComponent<HorizontalLayoutGroup>();
+            hlg.childForceExpandWidth = false;
+            hlg.childForceExpandHeight = false;
+            hlg.spacing = 2;
 
-            inventoryButton_go.AddComponent<Image>();
+            inventorySlot_go.AddComponent<Image>();
 
-            Button button = inventoryButton_go.AddComponent<Button>();
-            ColorBlock colorBlock = new ColorBlock();
-            colorBlock.normalColor = Color.white;
-            colorBlock.pressedColor = Color.blue;
-            button.colors = colorBlock;
             string localName = invName;
-            button.onClick.AddListener(
-                () => OnButtonClick(localName));
 
-            CreateTextComponent(inventoryButton_go, invName);
+            CreateTextComponent(inventorySlot_go, invName, TextAnchor.MiddleLeft);
+            CreateButtonComponents(inventorySlot_go, localName, new int[] { 1, 20, 50 });
```

Comment:
I thought of that, but this being a dev tool decided to leave the chance to go up to 50 if you want.  As it is, if you want to get a heap with maxStack, you just need to click a tile twice with the 50 option (the game will automatically resize it to the maxStack value in that case)